src/Control/Promise.purs

```diff
@@ -1,4 +1,4 @@
-module Control.Promise (fromAff, toAff, toAff', toAffE, Promise()) where
```

Comment:
I've been toying with renaming the module to `Web.Promise.Aff` too; any thoughts? Best practices for this sort of breaking change if so?

Reply:
not sure if that's appropriate or not, it would make sense but possibly steps on the toes of purescript-web / purescript-web-promise a bit. I'd want to get another opinion there.

I think this changeset is already a (minoary) breaking change though, the addition of a constraint is going to require changes in downstream code due to inference changes if I understand it?

Reply:
Correct; as far as I can tell the added constraint is only an issue if someone is trying to return a promise - which the constraint explicitly prevents, or if the type variable is left unbound as in the test case update.